metricflow-semantics/metricflow_semantics/query/query_resolver.py

```diff
@@ -425,31 +443,31 @@ def _resolve_query(self, resolver_input_for_query: ResolverInputForQuery) -> Met
         min_max_only_input = resolver_input_for_query.min_max_only
         apply_group_by_input = resolver_input_for_query.apply_group_by
 
+        mappings_to_merge: List[InputToIssueSetMapping] = []
+
+        # Resolve metrics.
```

Comment:
Needed to move this up - metrics must be resolved before building the `MetricFlowQueryResolutionPath` used later now that object syntax is supported. Previously, the metric names were already resolved since we only supported passing the name directly.